Procfile

```diff
@@ -0,0 +1 @@
+web: ./carnival
```

Comment:
Oh cool. I didn't know (or was told and didn't internalize) about the scheduler thing. That'd be better than what I'm doing in tee-io which is basically `forever $ run >> threadDelay`.